frameworks/cassandra/src/main/dist/cassandra.yaml

```diff
@@ -143,6 +143,18 @@ otc_coalescing_strategy: {{CASSANDRA_OTC_COALESCING_STRATEGY}}
 credentials_validity_in_ms: {{CREDENTIALS_VALIDITY_IN_MS}}
 credentials_update_interval_in_ms: {{CREDENTIALS_UPDATE_INTERVAL_IN_MS}}
 permissions_cache_max_entries: {{PERMISSIONS_CACHE_MAX_ENTRIES}}
+cdc_enabled: {{CASSANDRA_CDC_ENABLED}}
+prepared_statements_cache_size_mb: {{CASSANDRA_PREPARED_STATEMENTS_CACHE_SIZE_MB}}
+thrift_prepared_statements_cache_size_mb: {{CASSANDRA_THRIFT_PREPARED_STATEMENTS_CACHE_SIZE_MB}}
+column_index_cache_size_in_kb: {{CASSANDRA_COLUMN_INDEX_CACHE_SIZE_IN_KB}}
+slow_query_log_timeout_in_ms: {{CASSANDRA_SLOW_QUERY_LOG_TIMEOUT_IN_MS}}
+back_pressure_enabled: {{CASSANDRA_BACK_PRESSURE_ENABLED}}
```

Comment:
Should we template this ? If backpressure is disabled (ie not enabled) then we do not need to include the 
```
back_pressure_strategy:
  ...
```
configs right?

This also seems like it opens doors for customers to customize (like authn custom yaml block) _but_ this is outside the scope of this PR so we can leave that for another day if needed